starwars/src/App.js

```diff
@@ -27,6 +28,7 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <Cards characters={this.state.starwarsChars}/>
```

Comment:
Instead of creating a new component `Cards`, you could've mapped `this.state.starwarsChars` here:

```
<div className="cardContainer">
  {this.state.starwarsChars.map(character => {
    return (
      <StarCharacters
        key={`${character.name} ${character}`}
        starChars={character}
      />
    )
  })}
</div>
```